src/extra/others/fragment/lv_fragment.h

```diff
@@ -0,0 +1,338 @@
+/**
+ * Public header for Fragment
+ * @file lv_fragment.h
+ */
+
+#ifndef LV_FRAGMENT_H
+#define LV_FRAGMENT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../../lv_conf_internal.h"
+
+#if LV_USE_FRAGMENT
+
+#include "../../../core/lv_obj.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef struct _lv_fragment_manager_t lv_fragment_manager_t;
+
+typedef struct lv_fragment_t lv_fragment_t;
+typedef struct lv_fragment_class_t lv_fragment_class_t;
+typedef struct lv_fragment_managed_states_t lv_fragment_managed_states_t;
```

Comment:
For consistency please refactor `struct lv_fragment_...` to `struct _lv_fragment_...`